conf/functions_worker.yml

```diff
@@ -26,6 +26,13 @@ workerHostname: localhost
 workerPort: 6750
 workerPortTls: 6751
 
+# The metadata store URL
+# Examples:
+# * zk:my-zk-1:2181,my-zk-2:2181,my-zk-3:2181
+# * my-zk-1:2181,my-zk-2:2181,my-zk-3:2181 (will default to ZooKeeper when the schema is not specified)
+# * zk:my-zk-1:2181,my-zk-2:2181,my-zk-3:2181/my-chroot-path (to add a ZK chroot path)
+metadataStoreUrl: zk:localhost:2181
```

Comment:
what's the difference with the following `configurationMetadataStoreUrl`?

Reply:
metadataStoreUrl is used for  BookKeeperPackagesStorage#initializeDlogNamespace。If we want to start  BookKeeperPackagesStorage storage service  in a separately running workers. we must support this configuration.

Reply:
We can be back to use configurationMetadataStoreUrl if metadataStoreUrl is not set.

Reply:
@nlu90